internal/manifests/operator/grafana/datasource.go

```diff
@@ -0,0 +1,10 @@
+package grafana
+
+import (
+	grafanav1 "github.com/grafana-operator/grafana-operator/v5/api/v1beta1"
+	configv1alpha1 "github.com/grafana/tempo-operator/apis/config/v1alpha1"
+)
+
+func Datasource(featureGates configv1alpha1.FeatureGates, namespace string) *grafanav1. {
```

Comment:
Is this file still used?

Reply:
I am going to work on this file, so yes.
This PR is still in draft so things might change.